site/content/in-dev/unreleased/polaris-spark-client.md

```diff
@@ -128,3 +128,21 @@ The Polaris Spark client has the following functionality limitations:
 3) Rename a Delta table is not supported.
 4) ALTER TABLE ... SET LOCATION is not supported for DELTA table.
 5) For other non-Iceberg tables like csv, it is not supported.
+
+## Iceberg Spark Runtime Client compatibility with Polaris Spark Client
```

Comment:
side question: what is the Iceberg artifact called "runtime" but the corresponding Polaris artifact is a "client"?